.github/github_workflows.ex

```diff
@@ -193,7 +193,10 @@ defmodule GithubWorkflows do
       steps: [
         checkout_step(),
         [
-          uses: "superfly/flyctl-actions/setup-flyctl@master"
+          uses: "superfly/flyctl-actions/setup-flyctl@1.5",
+          with: [
+            version: "0.3.40"
```

Comment:
The `setup-flyctl` action recommends pinning the flyctl version for production releases, so I did so in addition to pinning the action version itself. This will need monitoring *separately* for upgrades.